config/locales/diaspora/en.yml

```diff
@@ -1091,18 +1091,23 @@ en:
 
   registrations:
     new:
+      title: "Join the community!"
+      subtitle: "You're about to create an account on %{podname}. One diaspora* account is enough to reach the whole network, you don't need to create a new account here if you already have one on another pod."
       enter_email: "Enter your email address"
-      enter_username: "Pick a username (only letters, numbers, and underscores)"
+      enter_username: "Pick a username (only letters, numbers, _ and -)"
       enter_password: "Enter a password (six character minimum)"
       enter_password_again: "Enter the same password as before"
       sign_up: "Create account"
       email: "Email"
+      email_advice: "Your e-mail will never be shared with anyone and will only be used to allow you to recover your account or send notifications."
       username: "Username"
+      username_advice: "Your username is unique and can't be changed. It allows others to find you in the network."
       password: "Password"
       password_confirmation: "Password confirmation"
       submitting: "Submitting..."
       terms: "By creating an account you accept the %{terms_link}."
       terms_link: "terms of service"
+      import_indication: "You already have a diaspora* account and you want to migrate it on %{podname}? Please sign up first and then import your previous account from the settings."
```

Comment:
It's difficult to reword this and keep it simple. This is my best effort so far: "If you want to migrate  an existing diaspora* account to  %{podname}, please sign up first and then go to the user settings page to import your previous account."

Reply:
Keeping the question makes it more dynamic imo. "You already have a diaspora* account and you want to migrate it on %{podname}? Please sign up first and then go to the user settings page to import your previous account."
Is that okay?

Reply:
""You already have a diaspora* account and you want to migrate it on %{podname}?" Just doesn't read well in English, I'm afraid. I can't think of a way of phrasing it well. Let me think again...

"Do you want to migrate an existing diaspora* account to %{podname}? If so, sign up first and then go to the user settings page to import your previous account."

That works. Does it seem dynamic enough to you?